api/lib/domain/usecases/create-or-update-certification-center-invitation.js

```diff
@@ -0,0 +1,27 @@
+import bluebird from 'bluebird';
+
+const createOrUpdateCertificationCenterInvitation = async function ({
+  certificationCenterId,
+  emails,
+  locale,
+  certificationCenterRepository,
+  certificationCenterInvitationRepository,
+  certificationCenterInvitationService,
+}) {
+  const certificationCenter = await certificationCenterRepository.get(certificationCenterId);
+
+  const uniqueEmails = [...new Set(emails)];
+  const trimmedUniqueEmails = uniqueEmails.map((email) => email.replace(/ /g, ''));
+
+  return bluebird.mapSeries(trimmedUniqueEmails, (email) => {
+    return certificationCenterInvitationService.createOrUpdateCertificationCenterInvitation({
+      certificationCenterInvitationRepository,
+    })({
+      certificationCenter,
+      email,
+      locale,
+    });
+  });
+};
```

Comment:
On peut eviter le return

```suggestion
  return bluebird.mapSeries(trimmedUniqueEmails, (email) =>
certificationCenterInvitationService.createOrUpdateCertificationCenterInvitation({
      certificationCenterInvitationRepository,
    })({
      certificationCenter,
      email,
      locale,
    });
};
```